flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/source/assigners/MySqlSnapshotSplitAssignerTest.java

```diff
@@ -207,28 +160,45 @@ public void testAssignTableWithMultipleKey() {
 
     @Test
     public void testAssignTableWithSparseDistributionSplitKey() {
-        // test table with sparse split key order like 0,10000,20000,3000 instead of 0,1,2,3
+        // test table with sparse split key order like 0,2,4,8 instead of 0,1,2,3
         List<String> expected =
                 Arrays.asList(
-                        "customer_card null [26317]",
-                        "customer_card [26317] [32633]",
-                        "customer_card [32633] [38949]",
-                        "customer_card [38949] [45265]",
-                        "customer_card [45265] null");
+                        "customers_3 null [10]", "customers_3 [10] [18]", "customers_3 [18] null");
```

Comment:
Could you add some new tests for the sparse/dense cases instead of updating the exsiting tests? It's hard to estimate whether the removed test is needed.

Reply:
The existing tests need to update after using the dynamic chunk size assignment